tests/behat/features/capabilities/group/group-create-flexible.feature

```diff
@@ -219,11 +220,22 @@ Feature: Create flexible Group
     Then I am on "all-groups"
     And I should not see "Flexible group - Secret option"
 
-    # Check this as an user with SM permissions.
+    # Check this as an user with LU permissions.
     Given I am logged in as an "authenticated user"
     When I am on "all-groups"
     And I should not see "Flexible group - Secret option"
 
+    # Check this as an user with SM permissions.
+    Given I am logged in as "SiteManagerOne"
+    When I am on "all-groups"
+    Then I should see "Flexible group - Secret option"
+    When I click "Flexible group - Secret option"
+    Then I should see the link "Stream"
+    And I should see the link "About"
+    And I should see the link "Members"
+    And I should not see the link "Events"
```

Comment:
@ronaldtebrake Are you sure that a SiteManger - should not see Events and Topics?